assets/src/design-system/components/dialog/stories/index.js

```diff
@@ -134,3 +135,59 @@ export const With2Actions = () => {
     </>
   );
 };
+
+export const With2ActionsDarkTheme = () => {
+  const [toggleDialog, setToggleDialog] = useState(false);
+
+  const ActionsNode = (
+    <>
+      <Button
+        size={BUTTON_SIZES.SMALL}
+        type={BUTTON_TYPES.TERTIARY}
+        onClick={() => {
+          action('cancel button clicked');
+          setToggleDialog(!toggleDialog);
+        }}
+      >
+        {'Secondary'}
+      </Button>
+      <Button
+        size={BUTTON_SIZES.SMALL}
+        type={BUTTON_TYPES.PRIMARY}
+        onClick={() => {
+          action('button clicked');
+        }}
+      >
+        {'Primary'}
+      </Button>
+    </>
+  );
+  return (
+    <DarkThemeProvider>
+      <Button
+        size={BUTTON_SIZES.SMALL}
+        type={BUTTON_TYPES.PRIMARY}
+        onClick={() => setToggleDialog(!toggleDialog)}
+      >
+        {'Toggle Dialog'}
+      </Button>
```

Comment:
Nit: should this have a dark background here to show what the dialog looks like against a dark color?
![image](https://user-images.githubusercontent.com/22185279/111219626-e13a8300-859d-11eb-9ce9-762ae33d4c6c.png)

Reply:
i think if anything, the light theme needs to get a dark background since these are actually inverted. Probably a good thing to add a note about in the design system.